File: DataBinding/Section/IListBinding/C#/IListBinding.DataLayer/Product.cs

namespace ActiveReports.Samples.IListBinding.DataLayer
{
	/// <summary>
	/// Summary description for Product.
	/// </summary>
	public class Product
	{
		internal Product()
		{
		}

		//******
		//The properties listed below define a product in the datalayer. 
		//******

		public int ProductID { get; set; }

		public string ProductName { get; set; }

		public int SupplierID { get; set; }

		public int CategoryID { get; set; }


		public string QuantityPerUnit { get; set; }

		public decimal UnitPrice { get; set; }

		public int UnitsInStock { get; set; }

		public int UnitsOnOrder { get; set; }

		public int ReorderLevel { get; set; }

		public bool Discontinued { get; set; }
	}
}

File: DataBinding/Section/IListBinding/VB.NET/IListBinding.DataLayer/Product.vb
'<summary>
' Summary description for Product.
'</summary>
Public Class Product
	Friend Sub New()
	End Sub 'New

	'******
	'* The properties listed below define a product in the datalayer. 
	'******

	Public Property ProductID As Integer

	Public Property ProductName As String

	Public Property SupplierID As Integer

	Public Property CategoryID As Integer

	Public Property QuantityPerUnit As String

	Public Property UnitPrice As Decimal

	Public Property UnitsInStock As Integer

	Public Property UnitsOnOrder As Integer

	Public Property ReorderLevel As Integer

	Public Property Discontinued As Boolean
End Class 'Product
